### ISSUE TITLE
"Related" aspect of `chemical` scares out new users

### ISSUE THREAD

**Issue by Adafede** [WDscholia/scholia#2518]
**What is the issue?**
I tried to demonstrate how useful Scholia can be for chemistry, for example showcasing https://scholia.toolforge.org/chemical/Q18216, but new users directly blocked on the first part where they saw data that scared them out, and then did not get any further, where the data is much clearer.
![Screenshot 2024-06-13 at 14 16 39](https://github.com/WDscholia/scholia/assets/44283913/1264b9be-1b4b-4670-a50d-0982900b5e80)

**Why is this a problem?**
It is a problem as newcomers might not understand why this data is shown there, and at first sight, makes little to no sense. The first content they see is not as "understandable" as all the rest below, which is sad.

**How could this be addressed?**
- [ ] Place the "related" aspect further down
- [ ] Add some clearer explanations on its origin without having to mouse over
- [ ] Remove the "related" aspect

**What are good places to discuss this?**
Here? 😊 

@egonw

**Comment by egonw** [comment]
Yeah, good point. For any Scholia page, I never understood the relatedness, but for chemicals what the relationship is essential. And it's not chemical, but on a `chemical` aspect it's perfectly understandable the user expect a chemical relationship. It should be moved down, indeed.

@fnielsen, @Daniel-Mietchen, we may need to reconsider this for other *specific* aspects too.

**Comment by fnielsen** [comment]
Over at Synia there are "Related researchers" at https://synia.toolforge.org/#author/Q20895241 and "Related lexemes" at Related lexemes. In Scholia we have also "related papers based on local cocitation network. These could be used for stealing/inspiration.

**Comment by Adafede** [comment]
There is already 

"Compounds with same connectivity" (https://scholia.toolforge.org/chemical/Q18216#related) funily already named "related", which also relates to #2484

I could think of the two ideas below for now:

`chemicals found in the same taxon as the current chemical`
```sparql
# tool: scholia
PREFIX target: <http://www.wikidata.org/entity/Q18216>

SELECT DISTINCT ?chemical WHERE {
  target: wdt:P703/^wdt:P703 ?chemical.
} 
  LIMIT 250
```

`chemicals belonging to the same chemical class as the chemical`
```sparql
# tool: scholia
PREFIX target: <http://www.wikidata.org/entity/Q2079986>

SELECT DISTINCT ?chemical WHERE {
  VALUES ?chemical_class {
        wd:Q15711994 # group of isomeric entities
        wd:Q17339814 # group or class of chemical substances
        wd:Q47154513 # structural class of chemical entities
        wd:Q55640599 # group of chemical entities
        wd:Q56256173 # class of chemical entities with similar applications or functions
        wd:Q56256178 # class of chemical entities with similar source or occurrence
        wd:Q55662456 # group of ortho, meta, para isomers
        # wd:Q59199015 # group of stereoisomers # TODO see later on when fast enough
        wd:Q72070508 # group or class of chemical entities
        wd:Q74892521 # imprecise class of chemical entities
    }
  ?class wdt:P31 ?chemical_class.
  target: wdt:P279 ?class.
  ?class ^wdt:P279 ?chemical.
} 
  LIMIT 250
```
